Dockerfile

```
@@ -1,15 +1,39 @@
-FROM python:3.10-bullseye
+FROM python:3.11-slim
 
-RUN mkdir /app
+WORKDIR /app
 
-COPY *.py /app/
-COPY requirements.txt /app/
-COPY .env /app/
+# Install system dependencies 
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)